internal/commands/app/create_inputs.go

```diff
@@ -228,11 +242,11 @@ func (i createInputs) args(omitDryRun bool) []flags.Arg {
 	if i.Environment != realm.EnvironmentNone {
 		args = append(args, flags.Arg{flagEnvironment, i.Environment.String()})
 	}
-	if i.Cluster != "" {
-		args = append(args, flags.Arg{flagCluster, i.Cluster})
+	if len(i.Clusters) > 0 {
+		args = append(args, flags.Arg{flagCluster, strings.Join(i.Clusters, ",")})
```

Comment:
also probably want to preserve the `!= ""` check somehow.  although now this likely depends on where we land about errors/warnings for any unresolved, specified data source names